package.json

```diff
@@ -91,8 +93,10 @@
     "expect-puppeteer": "4.4.0",
     "gettext-parser": "4.0.4",
     "husky": "6.0.0",
+    "jest-environment-jsdom": "27.5.1",
     "jsdoc": "3.6.7",
     "lint-staged": "10.2.11",
+    "markdown-table": "3.0.2",
```

Comment:
Same question if we're using `markdown-table`.

Reply:
I am going to remove it.